url_creator.py

```diff
@@ -0,0 +1,5 @@
+import os.path
+
+def url_creator(product_id):
+    url = 'https://www.amazon.com'
+    return os.path.join(url, product_id)
```

Comment:
this will result in 'https://www.amazon.com/product_id', but we want 'https://www.amazon.com/dp/product_id'